File: app/src/main/java/com/dream/bjst/account/ui/AccountSettingActivity.java
package com.dream.bjst.account.ui;

import android.view.View;

import com.dream.bjst.R;
import com.dream.bjst.base.BaseActivity;
import com.hjq.bar.TitleBar;
import com.ruffian.library.widget.RTextView;

public class AccountSettingActivity extends BaseActivity {
    TitleBar mTitleBar;
    RTextView versionTv;

    @Override
    protected int initLayout() {
        return R.layout.activity_account_setting;
    }

    @Override
    protected void initView() {
        mTitleBar=fvbi(R.id.account_my_setting_title);
        versionTv=fvbi(R.id.setting_version);
    }

    @Override
    protected void initData() {
        mTitleBar.getLeftView().setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                onBackPressed();
            }
        });
        //
        versionTv.setText(getVersionCode()+".0.0");

    }
}

File: app/src/main/java/com/dream/bjst/account/ui/AccountSettingActivity.kt
package com.dream.bjst.account.ui

import android.os.Bundle
import android.view.View
import com.blankj.utilcode.util.ActivityUtils
import com.dream.bjst.BuildConfig
import com.dream.bjst.account.vm.AccountViewModel
import com.dream.bjst.common.UserManager
import com.dream.bjst.databinding.ActivityAccountSettingBinding
import com.dream.bjst.login.LoginActivity
import com.tcl.base.common.ui.BaseActivity
import com.tcl.base.kt.ktClick
import com.tcl.base.kt.ktStartActivity

class AccountSettingActivity : BaseActivity<AccountViewModel, ActivityAccountSettingBinding>() {
    override fun initView(savedInstanceState: Bundle?) {
    }

    override fun initData() {
        //获取标题栏的返回按钮
        mBinding.accountMySettingTitle.leftView.setOnClickListener(View.OnClickListener { onBackPressed() })
        //获取安卓版本号
        mBinding.settingVersion.text = (BuildConfig.VERSION_NAME)
        //获取Email
        mBinding.accountEmail.text = UserManager.getCustomerEmail().ifEmpty { "ajsdklfjksldafa@gmil" }
        //设置用户姓名
        mBinding.accountName.text = UserManager.getUserName().ifEmpty { "User name" }
        //设置电话号码
        mBinding.accountPhone.text = UserManager.getUserPhone().ifEmpty { "8019872373"}
        event()

    }

    private fun event() {
        mBinding.logOutBtn.ktClick {
            UserManager.clearUserInfo()
            ktStartActivity(LoginActivity::class)
            ActivityUtils.finishAllActivities()
        }


    }

    override fun initDataOnResume() {
    }
}